Makefile

```diff
@@ -24,15 +41,18 @@ $(CRATES):
 $(WORKSPACES_WITH_RUST_MODULES):
 	$(MAKE) -C $@ install
 
+.PHONY: toolchain
+toolchain: ## install toolchain
+	scripts/toolchain/install-toolchain.sh
+
 .PHONY: deps
-deps: ## install dependencies
-# only use corepack on non-nix systems
-	[ -n "${NIX_PATH}" ] || corepack enable
-	command -v rustup && rustup update || echo "No rustup installed, ignoring"
+deps: ## install dependencies in CI
+	${YARN_ENVIRONMENT} yarn workspaces focus ${YARNFLAGS}
+# Don't fetch Rust crates since syncing with crates registry is slow
+ifeq ($(origin NO_CARGO),undefined)
```

Comment:
Isn't this obsolete now that we have vendored crates?

Reply:
absolutely, removed this flag